spec/rubocop/cop/rspec/base_spec.rb

```diff
@@ -1,25 +1,8 @@
 # frozen_string_literal: true
 
-RSpec.describe RuboCop::Cop::RSpec::Base do
-  subject(:cop) { RuboCop::RSpec::FakeCop.new(config) }
-
-  let(:config) do
-    rubocop_config =
-      {
-        'AllCops' => {
-          'RSpec' => {
-            'Patterns' => rspec_patterns
-          }
-        },
-        'RSpec/FakeCop' => {
-          'Exclude' => %w[bar_spec.rb]
-        }
-      }
-
-    RuboCop::Config.new(rubocop_config, 'fake_cop_config.yml')
-  end
-
-  let(:rspec_patterns) { ['_spec.rb$', '(?:^|/)spec/'] }
+RSpec.describe RuboCop::Cop::RSpec::Base, :config do
```

Comment:
I’m not sure I know if the `:config` metadata is necessary here. Can you explain?

Reply:
It comes from `rubocop/lib/rubocop/rspec/shared_contexts.rb` and defines the `cop` and `config` variables

Reply:
Actually, we mix `:config` metadata in for all the cops' specs, including even the `Base` abstract cop, so specifying it explicitly is not needed.
Removed.